goshopify.go

```diff
@@ -368,6 +379,40 @@ func (c *Client) doGetHeaders(req *http.Request, v interface{}) (http.Header, er
 	return resp.Header, nil
 }
 
+func (c *Client) logRequest(req *http.Request) {
+	if req == nil {
+		return
+	}
+	if req.URL != nil {
+		c.log.Debugf("%s: %s", req.Method, req.URL)
```

Comment:
Either use `req.URL.String()` or use the `%#v` or `%v` printf directive?

Reply:
great idea!